lib/Common/Memory/Recycler.inl

```diff
@@ -280,7 +296,27 @@ Recycler::RealAllocFromBucket(HeapInfo* heap, size_t size)
 #endif
         )
     {
-        VerifyZeroFill(memBlock + sizeof(FreeObject), sizeCat - (2 * sizeof(FreeObject)));
+        // TODO: looks the check has been done already
+
+        bool isPageHeapAlloc = false;
```

Comment:
this is for 'is pageheap enabled on the specified bucket', yes we can wrap this to recycler for given attributes and sizeCat

---
In reply to: [60989559](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60989559) [](ancestors = 60989559)